docs/content/en/docs/how-tos/buildpacks/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Building Artifacts with CNCF Buildpacks"
+linkTitle: "Buildpacks"
+weight: 100
+---
+
+This page describes building Skaffold artifacts with [buildpacks](https://buildpacks.io/).
+Buildpacks enable building language-based containers from source code, without the need for a Dockerfile.
+
+To use buildpacks with Skaffold, please install the following tools:
+
+* [pack](https://github.com/buildpack/pack)
+* [docker](https://www.docker.com/)
+
+Once installed, you must choose a buildpack image to build your artifacts.
+To see a list of available buildpacks, run:
+
+```shell
+$ pack suggest-builders
+```
+
+Choose a buildpack from the list, and set it with:
```

Comment:
Since these are meant to be general instructions, and the different builders support different languages, I figured people would need to look at the list to pick one that works for them. 

Later on in the tutorial I pick a specific builder which works with Go

Reply:
can we then instead just show the command to `suggest-builders` and skip 
When i was running this tutorial, i spent time looking for what other builders did and which one did i need to set. (which you mention later) Its a distraction not needed at this time.

Reply:
People will have to choose a builder for this to work though -- WDYT of moving the tutorial to the top? That way people can test the sample first, and then follow the general instructions if they decide to use buildpacks with their own projects.

Reply:
Just made that change, lmk if you feel it's easier to follow now.

Reply:
yes look better.